### ISSUE TITLE
Buffer issue

### ISSUE THREAD

**Issue by JackCA** [hone/heroku-cli-static#4]
I get the following error after I run `heroku static:deploy`:

```
buffer.js:106
      throw new Error(
      ^

Error: If encoding is specified then the first argument must be a string
    at new Buffer (buffer.js:106:13)
    at Readable.<anonymous> (/Users/janderson/.local/share/heroku/plugins/node_modules/archiver/lib/util/index.js:32:15)
    at emitNone (events.js:91:20)
    at Readable.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:926:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

I have tried multiple version of node 4.x

**Comment by JackCA** [comment]
Potentially related to https://github.com/heroku/heroku-builds/issues/19

**Comment by hone** [comment]
@JackCA released 1.0.1 which should fix this.

**Comment by JackCA** [comment]
:clap:
